### DISCUSSION TITLE
Vercel is having an outage

### DISCUSSION THREAD

**Discussion by hecktarzuli** [vercel/community#2528] [Bug Report]
After running a deploy this morning, random files are returning 500/502 from vercel.  I'd love to report this directly to Vercel, but apparently since I'm on a hobby account (for my little site), I'm resorted to community forums, even for outages.

https://deltener.com/
![image](https://github.com/vercel/community/assets/2112282/51da889b-027e-4fbe-b378-0a70bac4b0d4)
![image](https://github.com/vercel/community/assets/2112282/769b6120-3fbb-4101-b521-f7e411cc28b3)
![image](https://github.com/vercel/community/assets/2112282/0dbe1c87-49e4-4fb8-9205-564a80d28d26)

It's not just me.
https://twitter.com/JoshDeltener/status/1659926601448976388

Status shows all good (which it clearly is not)
![image](https://github.com/vercel/community/assets/2112282/01207a18-2f8b-42a9-9bae-14ccc75100a7)
![image](https://github.com/vercel/community/assets/2112282/ba110b43-5e21-4077-bef2-cfb497c954c6)

**Comment by amyegan** [comment]
Thanks for reporting this. It appears to be related to the incident [`Elevated INTERNAL_ROUTER_CANNOT_GET_PATH error rates`](https://www.vercel-status.com/incidents/mg87wqkq22nt), which has been resolved.
